### DISCUSSION TITLE
Plugin settings icon

### DISCUSSION THREAD

**Discussion by CodingPupper3033** [inventree/InvenTree#5775] [Q&A] [answered]
Hello, quick question - is there currently any way to add an icon for your plugin in settings?

Currently, all plugins have a dot. Are you able to change that to an icon of your choice? (I know it's a small detail)
![image](https://github.com/inventree/InvenTree/assets/37462274/6c8641a0-d735-4d20-abc3-cd8c82e82614)

**Comment by SchrodingersGat** [comment, marked as answer]
@CodingPupper3033 currently not possible. We may implement this in the new user interface (see pinned issue about moving to react)

**Reply by CodingPupper3033** [reply]
Thank you!

I do genuinely really appreciate your help these few weeks!
